wbmalert.go

```diff
@@ -66,26 +67,30 @@ func createInitialSnapshot(website *website, wg *sync.WaitGroup) {
 	content, err := getWebsiteAsString(website)
 	if err == nil {
 		website.Snapshot = content
-		log.Println("Created initial snapshot for " + website.Name)
+		log.Println("[INFO]", "Created initial snapshot for "+website.Name)
 	} else {
-		log.Println(err)
+		log.Println("[ERROR]", website.Name, err)
 	}
 	defer wg.Done()
 }
 
 func getWebsiteAsString(website *website) (string, error) {
 	request, err := http.NewRequest(http.MethodGet, website.Url, nil)
+	errorText := fmt.Sprintf("Error fetching %s", website.Name)
 	if err != nil {
-		return "Error", err
+		return errorText, err
 	}
 	resp, err := client.Do(request)
 	if err != nil {
-		return "Error", err
+		return errorText, err
+	}
+	if resp.StatusCode != 200 {
+		return errorText, fmt.Errorf("http status code error occured : %d", resp.StatusCode)
```

Comment:
This would throw an error for status codes such as `204` or `301`. Maybe restrict it to codes `>= 400`?

Reply:
done.